tests/test_camera.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+import datetime
+
+import mock
+
+from greenpithumb import camera
+
+
+class TestCamera(unittest.TestCase):
+
+    def test_capture(self):
+        folder_path = 'C:\\Users\\Jeet\\'
+        mock_local_clock = mock.Mock()
+        mock_camera = mock.Mock()
+        mock_local_clock.now.return_value = datetime.datetime(2016, 7, 23, 10,
+                                                              51, 9, 928000)
+        camera.Camera(folder_path, mock_local_clock, mock_camera).capture()
+        mock_camera.capture.assert_called_once_with(
+            'C:\\Users\\Jeet\\2016-07-23-10',
+            format='jpeg')
```

Comment:
Suggest using [`contextlib.closing`](https://docs.python.org/2.7/library/contextlib.html#contextlib.closing) here. It calls `close` automatically at the end of the `with` block, even if there's an exception.

That's what we'll want actual callers to use.

```python
        with contextlib.closing(camera.Camera(folder_path, mock_local_clock,
                                              mock_camera)) as camera_manager:
            camera_manager.capture()

        mock_camera.capture.assert_called_once_with(
            'C:\\Users\\Jeet\\2016-07-23-10',
            format='jpeg')
        mock_camera.close.assert_called()
```